src/accelerate/components/List.tsx

```diff
@@ -129,27 +186,38 @@ export default function List( props: Props ) {
 							}
 
 							return (
-								<tr key={ post.id }>
-									<td className="record-traffic">
-										<div className="traffic-bar" style={ { right: `${ 100 - ( post.views / maxViews * 100 ) }%` } }></div>{ compactMetric( post.views ) }
+								<tr key={post.id}>
+									<td className='record-thumbnail'>
+										{ post.thumbnail && (
+											<img src={ post.thumbnail } alt={ post.title } width="100" height="50" />
+										) }
 									</td>
 									<td className="record-name">
-										{ post.title }
-										{ ( post.type.name === 'xb' || post.type.name === 'wp_block' ) && (
-											<span className="tag">{ post.type.label }</span>
-										) }
+										<div className='record-name__type'>
+											{ post.type.label }
+										</div>
+										<div className='record-name__tag'></div>
+										<div className='record-name__title'>
+											<a href={ post.url || '' } target="_blank">{ post.title }</a>
+										</div>
+									</td>
+									<td className="record-traffic">
+										<div className="traffic-bar" style={ { right: `${ 100 - ( post.views / maxViews * 100 ) }%` } }></div>{ compactMetric( post.views ) }
 									</td>
 									<td className={ `record-lift score-${ lift && lift >= 0 ? 'pos' : 'neg' }` }>
 										{ !! lift && ! isNaN( lift ) && ( lift >= 0 ? '↑' : '↓' ) }
 										{ !! lift && ! isNaN( lift ) && compactMetric( parseFloat( lift.toFixed( 1 ) ) ) }
 									</td>
-									<td className="record-author">
-										<img alt="" className="record-avatar" src={ post.author.avatar } />
-										{ post.author.name }
-									</td>
-									<td className="record-links">
-										{ post.url && ( <a href={ post.url }>{ __( 'View', 'altis' ) }</a> ) }
```

Comment:
Why remove the view link? I thought those were on the design still. If not it's ok to question what's on the designs if your instincts are telling you it makes sense to keep them.

Reply:
They were not on the designs, and the view action can be triggered by clicking the post link, which I assume the design included, it's also what I'd expect as a user.

<img width="292" alt="image" src="https://user-images.githubusercontent.com/451892/177298702-4eeec66a-9144-4dd7-8b92-a25abd5fd147.png">